apps/webapp/Dockerfile.slim

```
@@ -0,0 +1,61 @@
+RUN cd packages/shared && pnpm run build
+
+# Build the webapp
+RUN pnpm run build --filter=webapp...
+
+# Runner stage
+FROM node:20-alpine AS runner
+WORKDIR /app
+
+# Don't run production as root
+RUN addgroup --system --gid 1001 nodejs
+RUN adduser --system --uid 1001 nextjs
```

Hadolint (2.12.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)